app/scripts/background.ts

```diff
@@ -3,26 +3,22 @@
 
 import { getSessionInfo, openLoginForm, uploadEmoji } from './sub_modules/slack'
 
-const onClickContextMenus = async (
-  info: chrome.contextMenus.OnClickData,
-  tab: chrome.tabs.Tab,
-) => {
-  if (!info.srcUrl) {
+const addEmojiToTeam = async (imageUrl: string | undefined, givenTeamName?: string) => {
```

Comment:
Please specify the type of return value.

Reply:
fixed.